accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/history/impl/AcHistoryServiceImpl.java

```diff
@@ -83,7 +83,7 @@ public void persistHistory(PersistableInstallationLogger installLog) {
 
             String mergedAndProcessedConfig = installLog.getMergedAndProcessedConfig();
             if (StringUtils.isNotBlank(mergedAndProcessedConfig)) {
-                JcrUtils.putFile(historyNode, "mergedConfig.yaml", "text/yaml",
+                JcrUtils.putFile(historyNode, "mergedConfig.txt", "text/plain",
```

Comment:
so this is a shame to have only a "toString" representation saved - isn't it possible to somehow generate the yaml without too much memory impact? (IIRC today snippets of this file can be used to test things and do some "round-tripping")

Reply:
suggestions welcome, SnakeYAML itself has huge overhead!

Reply:
I suppose we could fairly easily write a toString method, that creates valid yaml (instead of just using `ToStringBuilder`) - but we can also do that separately, not the most important to have immediately

Reply:
@ghenzler I've created #651 to improve toString. Hope it's fine.